build.gradle

```diff
@@ -66,7 +66,7 @@ ext.libraries = [
                 "org.junit.jupiter:junit-jupiter-engine:${versions.junit5}",
                 "org.junit.jupiter:junit-jupiter-params:${versions.junit5}",
         ],
-        mantisShaded   : "io.mantisrx:mantis-shaded:1.+",
+        mantisShaded   : "io.mantisrx:mantis-shaded:2.0.0-rc.1",
```

Comment:
Why an RC vs a release?

Reply:
I haven't released 2.0.0 yet. till then I at least want all deps to be 2.+ even if it's a rc release.

Reply:
Makes sense!